RELEASE_NOTES.md

```diff
@@ -0,0 +1,2 @@
+#### 1.0.0 May 30 2016 ####
```

Comment:
The build script is tagging the package with "-beta1". Open to suggestions about what the version should be, but I think we can update it with a PR just before releasing. I'm not planning to release (and retract the old versions) until after the TCK is merged in.